### PR TITLE
Feature rds module

### PR DESCRIPTION
**Pull request by lowkent2me**
RDS module completed

### CODE REVIEW

File: terraform/environments/prd/aws/rds_variables.tf [hunk 1]
```diff
@@ -0,0 +1,35 @@
+locals {
+omega_rds = {
```
Comment (ggramal) [inline terraform/environments/prd/aws/rds_variables.tf]: terraform fmt *

File: terraform/environments/prd/aws/vpc_variables.tf [hunk 2]
```diff
@@ -84,7 +84,37 @@ locals {
...
+            {
+              cidr   = "0.0.0.0/0"
+            }
+          ]
+        }
       }
     }
+
   }
+          subnet_rds = {
```
Comment (ggramal) [inline terraform/environments/prd/aws/vpc_variables.tf]: do we need this?

File: terraform/environments/prd/aws/rds_variables.tf (hunk 1 above)
Comment (lowkent2me) [reply terraform/environments/prd/aws/rds_variables.tf]: Fixed format

File: terraform/environments/prd/aws/vpc_variables.tf (hunk 2 above)
Comment (lowkent2me) [reply terraform/environments/prd/aws/vpc_variables.tf]: I forgot to remove this trash)

File: terraform/modules/aws/rds/main.tf [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  subnet_ids = var.rds_subnet_ids
+
+
+  tags = {
+    Name = "rds-subnet-tf"
+  }
+}
+
+# create the rds instance
+resource "aws_db_instance" "omega_db" {
```
Comment (ggramal) [inline terraform/modules/aws/rds/main.tf:41]: Please attach security group that you created to the instance

Comment (lowkent2me) [reply terraform/modules/aws/rds/main.tf:41]: Done

**Review by ggramal — APPROVED** [review]

**Pull request merged**